frontend/public/style/_overrides.scss

```diff
@@ -317,3 +317,8 @@ ul {
 .pf-v6-c-truncate--no-min-width {
   --pf-v6-c-truncate--MinWidth: 0 !important;
 }
+
+// Allow text wrapping in banner for mobile
+.pf-v6-c-banner {
```

Comment:
Nit that I missed the first time.  Move to line 104 so the rules are alphabetized by selector (I realize there are other alpha mistakes in this file, but no point in adding to it).